qa_dtv fails on s390x and powerpc

Debian build logs for 3.10.9.0-rc1 show that qt_dtv fails on the s390x and powerpc architectures:

https://buildd.debian.org/status/fetch.php?pkg=gnuradio&arch=s390x&ver=3.10.9.0%7Erc1-1&stamp=1702921730&raw=0
https://buildd.debian.org/status/fetch.php?pkg=gnuradio&arch=powerpc&ver=3.10.9.0%7Erc1-1&stamp=1702927887&raw=0

The error is identical on both architectures:
```
======================================================================
FAIL: test_000 (__main__.test_dtv.test_000)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/<<PKGBUILDDIR>>/gr-dtv/python/dtv/qa_dtv.py", line 171, in test_000
    self.assertFloatTuplesAlmostEqual(out_data, expected_data, 5)
  File "/<<PKGBUILDDIR>>/gnuradio-runtime/python/gnuradio/gr_unittest.py", line 97, in assertFloatTuplesAlmostEqual
    return all([
               ^
  File "/<<PKGBUILDDIR>>/gnuradio-runtime/python/gnuradio/gr_unittest.py", line 98, in <listcomp>
    self.assertAlmostEqual(x, y, places, msg)
AssertionError: -0.7144345 != 1.0423439e-10 within 5 places (0.7144345 difference)
```

So it would appear there's either a bug in the gr-dtv code, or else the qa_dtv test is flaky.

/cc @drmpeg

--- a/gr-dtv/python/dtv/qa_dtv.py
+++ b/gr-dtv/python/dtv/qa_dtv.py
@@ -166,7 +166,7 @@ class test_dtv(gr_unittest.TestCase):
         self.assertEqual(getsize(self.outfile), getsize(testfile))
 
         out_data = np.fromfile(self.outfile, dtype=np.float32)
-        expected_data = np.fromfile(testfile, dtype=np.float32)
+        expected_data = np.fromfile(testfile, dtype=np.dtype("<f"))
 
         self.assertFloatTuplesAlmostEqual(out_data, expected_data, 5)
 

